ghinsights/USQL/USQL.usqlproj

```diff
@@ -247,6 +247,7 @@
       <ClusterDatabase>GHInsights</ClusterDatabase>
       <ClusterSchema>dbo</ClusterSchema>
     </Script>
+    <Script Include="QueryDates.usql" />
```

Comment:
This project should only contain USQL scripts related to the pipeline.  Remove reference to QueryDates.usql and QueryDates.usql.cs.